packages/reactive_ble_mobile/lib/src/generated/bledata.pb.dart

```diff
@@ -10,26 +10,12 @@ import 'dart:core' as $core;
 import 'package:protobuf/protobuf.dart' as $pb;
 
 class ScanForDevicesRequest extends $pb.GeneratedMessage {
-  static final $pb.BuilderInfo _i = $pb.BuilderInfo(
```

Comment:
can you run `melos format` for the whole repo? I think this will drastically change the diff of this PR